### PR TITLE
git217 initial code

### PR DESCRIPTION
**Pull request by davidradl**
Signed-off-by: David Radley <david_radley@uk.ibm.com>

### CODE REVIEW

File: omrs-database-polling-repository-event-mapper/build.gradle [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Contributors to the ODPi Egeria project.
+ */
+
+// Artifact names are taken from the directory by default, set in settings.gradle to override
+// The 'name' for the maven artifact, and description are set here
+ext.name = 'Abstract database polling event mapper connector'
```
Comment (planetf1) [inline omrs-database-polling-repository-event-mapper/build.gradle:8]: The name will be used as the gradle artifact. I think we should set this to the desired short name - and figure out if you can follow the existing naming scheme (names are starting to get quite long)

The gradle publish could override this name, but I don't see a need for both?

File: omrs-database-polling-repository-event-mapper/build.gradle [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+description = 'Abstract database polling event mapper connector'
+ext {
```
Comment (planetf1) [inline omrs-database-polling-repository-event-mapper/build.gradle:10]: the egeria pom is brought in as a list of managed dependencies, so you should find versions there. I think by default gradle will resolve to the highest of there or here.

**Review by planetf1 — CHANGES_REQUESTED** [review]
initial comments for now

File: omrs-database-polling-repository-event-mapper/build.gradle (hunk 2 above)
Comment (davidradl) [reply omrs-database-polling-repository-event-mapper/build.gradle:10]: @planetf1  are you saying remove ```
jacksonVersion = '2.13.2'
    jupiterVersion = '5.8.2'
    springwebVersion = '5.3.19' 
```and not use those variables?

File: omrs-database-polling-repository-event-mapper/build.gradle (hunk 1 above)
Comment (davidradl) [reply omrs-database-polling-repository-event-mapper/build.gradle:8]: the other connectors seem to use natural names - are you suggesting a shortened hyphenated name for this?

File: omrs-database-polling-repository-event-mapper/build.gradle (hunk 2 above)
Comment (davidradl) [reply omrs-database-polling-repository-event-mapper/build.gradle:10]: II have removed these variables

**Comment by planetf1** [comment]
Proposed patch (on top of 6681d9fb96f44c51ddd7bbd15cdb0a25440075f3 )
[0001-217-Fix-up-compile-issues-with-omrs-database-polling.patch.gz](https://github.com/odpi/egeria-database-connectors/files/9936332/0001-217-Fix-up-compile-issues-with-omrs-database-polling.patch.gz)

(uncompress first)

File: omrs-database-polling-repository-event-mapper/src/main/java/org/odpi/egeria/connectors/databasepollingrepository/eventmapper/OMRSDatabasePollingRepositoryEventMapper.java [hunk 3]
```diff
@@ -438,7 +450,7 @@ private void getRequiredTypes() throws ConnectorCheckedException {
                 for (String typeName : SupportedTypes.supportedTypeNames) {
 
                     TypeDef typeDef = repositoryHelper.getTypeDefByName("OMRSDatabasePollingRepositoryEventMapper",
-                            typeName);
+                                                                        typeName);
```
Comment (planetf1) [inline omrs-database-polling-repository-event-mapper/src/main/java/org/odpi/egeria/connectors/databasepollingrepository/eventmapper/OMRSDatabasePollingRepositoryEventMapper.java:453]: oops - sorry looks like I may have hit the space bar in error! formatting?

**Pull request merged**